script/upload

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-REPO="git-lfs/git-lfs"
+REPO="git-lfs/git-lfs-ci-test"
```

Comment:
Is this change from commit a46bc9d9770587f974c5ee5a6e4582380287a8a4 (with the commit message "WIP" and no description) still desired?

Reply:
No, that should be dropped.  I've done so.